apps/admin-ui/src/spa/application-rounds/[id]/review/AllocatedEventsTable.tsx

```diff
@@ -60,10 +60,11 @@ function timeSlotMapper(t: TFunction, slot: Node): ApplicationScheduleView {
   const applicationPk = application.pk ?? 0;
   const reservationPk = slot.recurringReservation?.reservations[0]?.pk ?? null;
   const link = getReservationUrl(reservationPk);
-  const accessCodeActiveAlert = slot.recurringReservation
-    ?.isAccessCodeIsActiveCorrect
-    ? ""
-    : t("RequestedReservation.accessCodesNotActive");
+  const accessCodeActiveAlert =
```

Comment:
Should it be `slot.rr?.shouldHaveActiveAccessCode && ! slot.rr?.isAccessCodeIsActiveCorrect ? t(...) : ""`

Reply:
That's basically what we're doing now - or the check result wouldn't change.

Reply:
..but sure, I can change it :D It might be 2% more readable the way you suggest. :D

Reply:
But am I too hungry to think straight or:
shouldHaveActiveAccessCode == **false** &&
isAccessCodeIsActiveCorrect == true/false
=> "RequestedReservation.accessCodesNotActive"